app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1913,23 +1913,14 @@ public void addBooks(List<LibraryNetworkEntity.Book> books) {
     booksAdapter.notifyDataSetChanged();
   }
 
-  private void searchFiles() {
-    if (Build.VERSION.SDK_INT >= VERSION_CODES.M) {
-      if (ContextCompat.checkSelfPermission(this,
+   private void searchFiles() {
+    if (Build.VERSION.SDK_INT >= VERSION_CODES.M && ContextCompat.checkSelfPermission(this,
           Manifest.permission.READ_EXTERNAL_STORAGE)
           != PackageManager.PERMISSION_GRANTED) {
-        if (ActivityCompat.shouldShowRequestPermissionRationale(this,
-            Manifest.permission.READ_EXTERNAL_STORAGE)) {
-          Toast.makeText(this, R.string.request_storage,
-              Toast.LENGTH_LONG).show();
-        }
         ActivityCompat.requestPermissions(this,
             new String[] { Manifest.permission.READ_EXTERNAL_STORAGE },
             REQUEST_READ_STORAGE_PERMISSION);
-      } else {
-        fileSearch.scan(sharedPreferenceUtil.getPrefStorage());
-      }
-    } else {
+      }   else {
```

Comment:
There should be a single space before `else`. If you would have autoformatted you could have fixed all these at once.